src/main/index.ts

```diff
@@ -723,6 +723,25 @@ const router = t.router({
       ? await getTokenOrThrow(DEVHUB_APP_ID, NO_DEVHUB_AUTHENTICATION_TOKEN_FOUND)
       : undefined;
 
+    // example of fetching allowlist
+    // ------------------------------------------------
```

Comment:
This is the idea of how to fetch the list but it should probably happen in the frontend in the end.